DependencyInjection/Configuration.php

```diff
@@ -28,9 +28,9 @@ public function getConfigTreeBuilder()
         $rootNode
             ->children()
                 ->scalarNode('dir_name')->defaultValue('%kernel.root_dir%/DoctrineMigrations')->cannotBeEmpty()->end()
-                ->scalarNode('namespace')->defaultValue('Application\Migrations')->cannotBeEmpty()->end()
+                ->scalarNode('namespace')->defaultValue('App\Migrations')->cannotBeEmpty()->end()
                 ->scalarNode('table_name')->defaultValue('migration_versions')->cannotBeEmpty()->end()
-                ->scalarNode('name')->defaultValue('Application Migrations')->end()
+                ->scalarNode('name')->defaultValue('App Migrations')->end()
```

Comment:
I don't really see the point of changing the name here ?

Reply:
Simply for some semblance of reader's consistency with SF4's usage of "App" -- and pointedly NOT using 'old' "Application".

Nbd either way -- it's more the namespace that's at issue.